docs/scarpet/api/Auxiliary.md

```diff
@@ -667,6 +667,9 @@ Available options in the scarpet app space:
   * `world_gamerules` - returns all gamerules in a map form (`rule`->`value`). Like carpet rules, values are returned as strings, so you can use appropriate value conversions using `bool()` or `number()` to convert them to other values. Gamerules are read-only to discourage app programmers to mess up with the settings intentionally applied by server admins. Isn't that just super annoying when a datapack messes up with your gamerule settings? It is still possible to change them though using `run('gamerule ...`.
   * `world_spawn_point` - world spawn point
   * `world_time` - Returns dimension-specific tick counter.
```

Comment:
this one is fine - I think all dimensions share the same counter anyways, so its not dimension specific